mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -876,6 +876,21 @@ linux_node_config {
 
 * `node_group` - (Optional) Setting this field will assign instances of this pool to run on the specified node group. This is useful for running workloads on [sole tenant nodes](https://cloud.google.com/compute/docs/nodes/sole-tenant-nodes).
 
+* `advanced_machine_features` - (Optional) Specifies options for controlling
+  advanced machine features.
+
+<a name="nested_advanced_machine_features"></a>The `advanced_machine_features` block supports:
+
+* `threads_per_core` - (Optional) The number of threads per physical core. To disable simultaneous multithreading (SMT) set this to 1. If unset, the maximum number of threads supported per core by the underlying processor is assumed.
+
+<a name="nested_network_config"></a>The `network_config` block supports:
```

Comment:
I think this line down is a copy-paste error?

Reply:
Removed this, seems like this was moved to another file and I should've remove it during a sync.